<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="wks-samira-n" name="com.calsoft.nexenta.vcp.SikuliTest" tests="2" failures="1" timestamp="20 Jul 2016 11:30:53 GMT" time="77.011" errors="0">
  <testcase name="testVCP" time="67.225" classname="com.calsoft.nexenta.vcp.SikuliTest"/>
  <testcase name="testVCP" time="9.786" classname="com.calsoft.nexenta.vcp.SikuliTest">
    <failure type="java.lang.RuntimeException" message="java.lang.reflect.InvocationTargetException">
      <![CDATA[java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at com.calsoft.nexenta.vcp.SikuliTestBase.handleException(SikuliTestBase.java:44)
	at com.calsoft.nexenta.vcp.SikuliTestBase.invokeMethodsInSequence(SikuliTestBase.java:36)
	at com.calsoft.nexenta.vcp.SikuliTestBase.invokeMethod(SikuliTestBase.java:26)
	at com.calsoft.nexenta.vcp.SikuliTest.testVCP(SikuliTest.java:72)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
Caused by: java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.calsoft.nexenta.vcp.SikuliTestBase.invokeMethodsInSequence(SikuliTestBase.java:33)
	... 26 more
Caused by: java.lang.RuntimeException: Error occured while clicking on the element E:\Nexenta\updated_Source\SourceTree\Sikuli_Gui\sikuli/src/test/resources/Images/TestImages/AddNexentaConfig1.PNG because java.lang.RuntimeException: Expected image not found
	at com.calsoft.nexenta.vcp.SikuliTestMethods.click_Image(SikuliTestMethods.java:97)
	... 31 more
]]>
    </failure>
  </testcase> <!-- testVCP -->
</testsuite> <!-- com.calsoft.nexenta.vcp.SikuliTest -->
